rdrf/rdrf/admin.py

```diff
@@ -347,19 +418,43 @@ def groups(self, obj):
     groups.short_description = _("Allowed Groups")
 
 
+class RegistryRegistrationNotificationFilter(admin.SimpleListFilter):
+    title = _("Registry registration notifications")
+    parameter_name = 'registry_registration'
+
+    def lookups(self, request, model_admin):
+        return [(reg.code, reg.name) for reg in Registry.objects.all()]
+
+    def queryset(self, request, queryset):
+        if self.value():
+            registry = Registry.objects.get(code=self.value())
+            return queryset.filter(registry=registry, description__in=EventType.REGISTRATION_TYPES)
+
+
 class EmailNotificationAdmin(admin.ModelAdmin):
     model = EmailNotification
-    list_display = ("description", "registry", "email_from", "recipient", "group_recipient")
+    list_display = ("description", "registry", "email_from", "recipient", "group_recipient", "disabled", "registry")
+    list_filter = (RegistryRegistrationNotificationFilter, )
 
     def get_changeform_initial_data(self, request):
         from django.conf import settings
         return {'email_from': settings.DEFAULT_FROM_EMAIL}
 
 
+def default_and_lang(email_template):
+    if email_template.default_for_notification:
+        return f"{email_template.default_for_notification} ({email_template.language})"
+    else:
+        return "-"
+
+
+default_and_lang.short_description = "Default for notification"
```

Comment:
Can you please put these on the class ie. `EmalTemplateAdmin` instead?

Also, `short_description` isn't translated.

Reply:
I attempted this but you can't refer to `self` in the class variables. I'll try add it as an instance variable in `__init__`.

Reply:
Or maybe a lambda :)